tests/StoringGroceries.tex

```diff
@@ -32,55 +36,83 @@ \subsection{Focus}
 	\end{figure}
 \end{minipage}
 
-
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Task
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Task}
 \begin{enumerate}
-	\item \textbf{Opening door:} The robot starts opening the Cupboard's door. If the robot is unable to open the door, it may ask the Referee to do it instead.
-	\item \textbf{Cupboard inspection:} The robot inspects the cupboard locating and categorizing existing groceries.
-	\item \textbf{Finding the table:} The robot turns around and locates the table.  
-	\item \textbf{Table inspection:} The robot approaches the table starts analyzing the newly bought groceries (i.e. objects).
-	\item \textbf{Moving objects:} The robot chooses which object to move first from the Table to the Cupboard, allocating similar objects all together.
+	\item \textbf{Evaluating the situation:} The robot inspects its surrounding and analyzing the best course of action. In any order, the robot has to:
+	\begin{itemize}
+		\item \textit{Inspect the cupboard} (locating and categorizing existing groceries).
+		\item \textit{Open the cupboard's door.} If the robot can't open the door, it may ask the Referee to do it.
+		\item \textit{Find the table}
+		\item \textit{Inspect the table} (analyze the newly bought groceries, i.e. objects).
+	\end{itemize}
+
+	\item \textbf{Moving objects:} The moves as many objects as possible from the Table to the Cupboard (max 5), allocating similar objects all together.
```

Comment:
First "as many as possible' and then "max 5", that's a contradiction

Reply:
You and @airglow suggested that change, so goes in.

Reply:
Fixed. Now is as many as possible within the given time but only first 5 scores. 
In any case, no robot so far is that fast. But assuming we get The Robo-Flash, additional objects would go for Outstanding Performance and to please the audience.